### ISSUE TITLE
Separate to per "type" packages

### ISSUE THREAD

**Issue by jelen07** [umpirsky/locale-list#4]
Is there any plan to separate this package "type" specific packages? Like

- locale-list-yaml
- locale-list-csv
- ...

The whole repository has ~179MB. When building an image with ~40MB, app ~10MB this is so much overhead.

**Comment by umpirsky** [comment]
What kind of "image" are you building?

**Comment by jelen07** [comment]
Oh, I'm sorry - docker image with the application that depends on this package.

**Comment by umpirsky** [comment]
I didn't plan to, I always assumed disk is cheap. 

But you can easily create composer post install script that cleans up what you don't need.

**Comment by jelen07** [comment]
Yes, it's cheap. But when you have more than 100 projects, which exists in x version (due to potential deployment rollback) you are trying to keep the vendor size as small as possible.
In the example above we're talking about more than 15 GB which must be hosted/transferred because of one library.
We could remove the files via post-install script, **. dockerignore** _(edited)_ or some other method but that means that we will be solving problem on a different level than the problem really is.
I think that the fork will be the most suitable.

**Comment by umpirsky** [comment]
I don't think .gitignore will help. Post install script sounds reasonable. If you go for the fork, you need to maintain it yourself.

**Comment by jelen07** [comment]
Sorry, I wanted to write ".dockerignore".
Adding a post-install script means adding extra complexity to the project. In the case of a fork, there is extra maintenance work that should be done to keep things up to date. But the project is cleaner.

**Comment by umpirsky** [comment]
I still think post-install script is your best option, but whatever works for you.
